.github/workflows/master-pushes.yml

```
@@ -4,9 +4,15 @@ on:
     branches: [master]
 
 jobs:
+  cache-pnpm-store:
+    name: Install everything PNPM and cache it
```

I think this makes more sense right?
```suggestion
    name: Install everything with PNPM and cache it
```

---

Bah, I missed this! Will fix and push